sdk/storage/azure-storage-blob-batch/README.md

```diff
@@ -22,7 +22,7 @@ definition, such as text or binary data.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-batch</artifactId>
-  <version>12.8.0</version>
+  <version>12.9.0-beta.1</version>
```

Comment:
we generally, keep README at stable versions, so existing and new customers try the stable release.

Reply:
Fixed